modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/policies/pages/components/Activities/TableListMobile.tsx

```diff
@@ -0,0 +1,163 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayTable from '@clayui/table';
+import classnames from 'classnames';
+import React, {useEffect, useState} from 'react';
+
+import Panel from '../PolicyDetail/Panel';
+import dataActivities from './DataActivities';
+
+import './index.scss';
+
+type Props = {
+	headers: TableHeaders[];
+	rows: InfoRowContent[];
+};
+
+type TableHeaders = {
+	bold?: boolean;
+	key: string;
+	value: string;
+};
+
+const {Body, Cell, Head, Row} = ClayTable;
+
+type InfoRowContent = {[keys: string]: string};
+
+const TableListMobile: React.FC<Props> = ({headers, rows}) => {
+	const [isPanelExpanded, setIsPanelExpanded] = useState<boolean[]>([]);
+	const [isRowSelected, setIsRowSelected] = useState<boolean>(false);
+
+	const ContentDescription = ({
+		columnOne,
+		columnTwo,
+	}: {
+		columnOne: string;
+		columnTwo: string;
+	}) => (
+		<div className="d-flex justify-content-between ml-3 my-3">
+			<div className="d-flex font-table ml-2 mr-9">{columnOne}</div>
+
+			<div>{columnTwo}</div>
+		</div>
+	);
+
+	const toggleSelectedRow = (index: number) => {
+		isPanelExpanded[index] === false
+			? setIsRowSelected(true)
+			: setIsRowSelected(false);
+	};
+
+	const resetExpandedPanel = dataActivities.map(() => {
+		return false;
+	});
+
+	const displayHistoryPanel = (index: number) => {
+		const supportArray = [...isPanelExpanded];
+		toggleSelectedRow(index);
+
+		if (isRowSelected === isPanelExpanded[index]) {
+			supportArray[index] = !supportArray[index];
+			setIsPanelExpanded(supportArray);
+		}
+		else {
+			resetExpandedPanel[index] = !resetExpandedPanel[index];
+			setIsPanelExpanded(resetExpandedPanel);
+		}
+	};
+
+	useEffect(() => {
+		setIsPanelExpanded(resetExpandedPanel);
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, []);
+
+	return (
+		<>
+			<div className="bg-neutral-0 w-100">
```

Comment:
Try to create a mother css class for this component.

Reply:
Could I create in the last PR?